client/src/style.css

```diff
@@ -1,11 +1,32 @@
+body{
+  font-family: 'Poppins', sans-serif;
+  color: rgb(70, 70, 70);
+}
```

Comment:
Got no idea what all of your styling does, but it looks legit. I'd like a rundown of it at some point